Description: add getpass() prototype
 This function has been deprecated and is not declared with
 _XOPEN_SOURCE>=600. To avoid fiddling too much with feature flags, we
 just add its prototype to ensure a correct declaration.
Forwarded: no (dead upstream)
Bug: #795879

Index: curlftpfs-0.9.2/ftpfs.c
===================================================================
--- curlftpfs-0.9.2.orig/ftpfs.c
+++ curlftpfs-0.9.2/ftpfs.c
@@ -36,6 +36,8 @@
 
 #define MAX_BUFFER_LEN (300*1024)
 
+extern char *getpass(const char *prompt);
+
 struct ftpfs ftpfs;
 static char error_buf[CURL_ERROR_SIZE];
 
